src/wctez.ml

```diff
@@ -0,0 +1,205 @@
+(* FA2 wrapper for ctez to allow it to integrate with Checker *)
+open Fa2Interface
+open Fa2Ledger
+open Fa12Interface
+open Common
+open Error
+
+(*****************************************************************************)
+(**                          {1 WRAPPER TYPES}                               *)
+(*****************************************************************************)
+
+type wctez_state =
+  { fa2_state : fa2_state;
+    ctez_fa12_address : Ligo.address;
+  }
+
+(** Token id for wrapped ctez tokens. *)
+let[@inline] wctez_token_id : fa2_token_id = Ligo.nat_from_literal "0n"
+
+(*
+(** Number of decimal digits for wctez tokens, identical to that for ctez. *)
+(* NOTE: Currently unused. *)
+let[@inline] wctez_token_decimal_digits = Ligo.nat_from_literal "6n"
+*)
```

Comment:
Ah yes, same as with `tezWrapper.ml`. We have to discuss at some point how to make the metadata of our two wrappers available on chain. Currently only Checker advertises the metadata of `kit` and `lqt` but the two wrappers should probably do the same as well.